app/assets/javascripts/faye-authentication.js

```diff
@@ -96,7 +96,7 @@ FayeAuthentication.prototype.authentication_required = function(message) {
       try {
         return (!this._options.whitelist(subscription_or_channel));
       } catch (e) {
-        this.logger.error("Error caught when evaluating whitelist function : " + e.message);
+        this.logger.error("[Faye] Error caught when evaluating whitelist function : " + e.message);
```

Comment:
added the [Faye] tag back because specs were expecting it, tested and indeed we didn't have the tag in the error, so added it here and there.